CHANGELOG-developer.next.asciidoc

```diff
@@ -88,6 +88,7 @@ The list below covers the major changes between 7.0.0-rc2 and main only.
 
 ==== Added
 
+- Add new metricset in Kubernetes module, `namespace_state`. {pull}36406[36406]
```

Comment:
I believe this comment https://github.com/elastic/beats/pull/36406#discussion_r1305346773 was related only to the field name, where metrics are store `kubernetes.state_namespace.*` - > `kubernetes.namespace_state.*` , not to the name of the metricset itself.

I think that the `state_*` in the metricset name is a reference to the metrics source - that they are scraped from the `kube_state_metrics`, if it is correct module name should be `state_namespace` for consistency. @MichaelKatsoulis @ChrsMark is my assumption regarding naming correct?

Reply:
Thanks @tetianakravchenko, you are right, I misunderstood. So now the document looks like this:

```json
{
    "@timestamp": "2019-03-01T08:05:34.853Z",
    "event": {
        "dataset": "kubernetes.namespace_state",
        ...
    },
    "kubernetes": {
        "namespace": "kube-node-lease",
        "namespace_state": {
            ...
        }
    },
    "metricset": {
        "name": "state_namespace",
        ...
    },
    ...
}
```

I could not change the `event.dataset`, I am not sure what this field means and elastic docs are not very clear either. I am guessing it is the "family" of fields that caused the metrics on the document.

Reply:
I think `event.dataset` should be `kubernetes.namespace`. I explain this at https://github.com/elastic/beats/pull/36406#discussion_r1311267016. I'm not sure if that line there also has impact in other places but `event.namespace` should be `kubernetes.namespace` if we want to be more accurate.